### ISSUE TITLE
Possible to add hints as to what stations will sound like?

### ISSUE THREAD

**Issue by corysimmons** [cheshire137/soma-chrome#32]
It'd be nice to not have to go to https://soma.fm to see what a station will sound like:

![image](https://cloud.githubusercontent.com/assets/224636/16277263/39f6e2ce-3880-11e6-8177-9f9c821142b1.png)

Is it asking too much to add these hints (on hover or a little clickable info icon) as to what a station will sound like?

If you're interested in something like this I could probably help by converting the select box to a div dropdown and adding in the relevant descriptions. If not, no biggie, still a cool extension. 👏

**Comment by cheshire137** [comment]
I agree! Beyond just station descriptions, showing some artist names like on http://somafm.com/groovesalad/artists.html would be helpful. Hat tip to @rustyhodge for pointing me at this page.

> I could probably help by converting the select box to a div dropdown

I have the station descriptions as tooltips in [HuxleyFM](https://github.com/cheshire137/huxleyfm) in a div menu instead of a select menu, so the code could be copied from there, if you want.

**Comment by corysimmons** [comment]
tbh, I listen to Soma because I don't know any of the musicians so the station descriptions would be better for me anyway. I'm also not sure what the UI of a huge musician list in a small Chrome extension would look like.

I didn't realize there was a desktop app for this. I'd definitely switch if there was a way to minimize it to the taskbar, but I suppose that should be it's own issue/request.
